Sources/Recycling/ListViews/AdManagement/StatisticsItemView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Copyright © FINN.no AS. All rights reserved.
+//
+
+import UIKit
+
+public class StatisticsItemView: UIView {
+    private lazy var imageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.contentMode = .scaleAspectFit
+        return view
+    }()
+
+    private lazy var valueLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 1
+        label.font = UIFont.title2 // subject to change medium/26 seems closer to the sketches
+        label.textColor = .licorice
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var textLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.font = UIFont.caption
+        label.textColor = .licorice
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var leftSeparator: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .sardine
+        return view
+    }()
+
+    private lazy var rightSeparator: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .sardine
+        return view
+    }()
+
+    var shouldShowLeftSeparator: Bool = false {
+        didSet { leftSeparator.isHidden = !shouldShowLeftSeparator }
+    }
+    var shouldShowRightSeparator: Bool = false {
+        didSet { rightSeparator.isHidden = !shouldShowRightSeparator }
+    }
+
+    public var itemModel: StatisticsItemModel? {
+        didSet {
+            guard let model = itemModel else { return }
+
+            let formatter = NumberFormatter()
```

Comment:
Can we make this `formatter` a property instead?

Reply:
I think it can also be a `static let` because formatters are pretty expensive to create.